vms/platformvm/warp/validator.go

```diff
@@ -52,6 +57,13 @@ func GetCanonicalValidatorSet(
 	pChainHeight uint64,
 	subnetID ids.ID,
 ) ([]*Validator, uint64, error) {
+	// There is a ~(1/canonicalValidatorSetSlowdownProbabilityDenominator) chance that we will output a 0 when calling this random number generator
+	// We can implement a slowdown for canonicalValidatorSetTimeSlowdownSeconds that occur randomly
+	// This will introduce real world conditions needed for the warp load test.
+	randomNum := rand.Intn(canonicalValidatorSetSlowdownProbabilityDenominator)
+	if randomNum == 0 {
+		time.Sleep(canonicalValidatorSetTimeSlowdownSeconds)
```

Comment:
I would recommend having some jitter here, maybe we can do something where we have a min + max delay? Instead of doing a `x` second wait randomly `1/y * 100` percent of the time, we can always do a `random(min, max)` delay.

Reply:
Yeah I like the idea of  introducing jitter. I feel like keeping the probability in case the load testers want to play with it. Thanks for the suggestion